src/components/PrimaryNav/PrimaryNav.module.css

```diff
@@ -61,5 +63,8 @@
  * Primary nav icon
  */
 .primary-nav__icon {
+  height: var(--eds-size-2-and-half);
+  width: var(--eds-size-2-and-half);
+  fill: var(--eds-theme-color-icon-neutral-default-inverse);
```

Comment:
`color` and `size` props should be updated in `PrimaryNavItem.tsx` instead, when calling `<Icon />`